bodhi/server/models.py

```diff
@@ -1149,6 +1153,10 @@ class Update(Base):
 
     user_id = Column(Integer, ForeignKey('users.id'))
 
+    # Greenwave
+    greenwave_policies_satisfied = Column(Boolean, default=True, nullable=False)
```

Comment:
I do not think this filed suits for Greenwave as Greenwave can't tell anything about "waiting","skipped", etc.